### PR TITLE
fix website_blog edition too slow & edition on hr, br

### PR DESCRIPTION
**Pull request by Gorash**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/static/src/js/editor/editor_menu.js [hunk 1]
```diff
@@ -136,12 +134,14 @@ var EditorMenu = Widget.extend({
...
             .filter(function () {
-                return !$(this).closest('.o_not_editable').length;
+                var $parent = $(this).closest('.o_editable, .o_not_editable');
+                return !$parent.length || $parent.hasClass('o_editable');
             })
             .not('link, script')
             .not('[data-oe-readonly]')
             .not('img[data-oe-field="arch"], br[data-oe-field="arch"], input[data-oe-field="arch"]')
             .not('.oe_snippet_editor')
+            .not('hr, br, input, textarea')
```
Comment (VincentSchippefilt) [inline addons/website/static/src/js/editor/editor_menu.js]: add image ? to check please

File: addons/website/static/src/js/editor/wysiwyg_multizone.js [hunk 2]
```diff
@@ -89,7 +89,7 @@ var WysiwygMultizone = Wysiwyg.extend({
                 }
             }
             if (ev.key.length === 1) {
-                this._onChange();
+                this.__onChange();
```
Comment (VincentSchippefilt) [inline addons/website/static/src/js/editor/wysiwyg_multizone.js]: rename to _onChangeThrottled

File: addons/website/static/src/js/menu/translate.js [hunk 3]
```diff
@@ -26,6 +26,9 @@ var TranslatePageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         });
         this._mustEditTranslations = context.edit_translations;
         if (this._mustEditTranslations) {
+            var url = window.location.href.replace(/([?&])&*edit_translations[^&]*&?/, '\$1');
```
Comment (VincentSchippefilt) [inline addons/website/static/src/js/menu/translate.js]: this also removes the # on the query string (2 times)

File: addons/website/static/src/js/menu/edit.js [hunk 4]
```diff
@@ -40,6 +40,8 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
             },
         });
         this._editorAutoStart = (context.editable && window.location.search.indexOf('enable_editor') >= 0);
+        var url = window.location.href.replace(/([?&])&*enable_editor[^&]*&?/, '\$1');
```
Comment (VincentSchippefilt) [inline addons/website/static/src/js/menu/edit.js]: this also removes the # on the query string (2 times)

File: addons/website/static/src/js/editor/editor_menu.js (hunk 1 above)
Comment (Gorash) [reply addons/website/static/src/js/editor/editor_menu.js]: I check, it's good. https://github.com/odoo/odoo/pull/31183/commits/f857610c433b2b95168155c4cd804700b40faad5#diff-788a9b71279ba1f6a0dc4534569e9dc4R201

File: addons/web_editor/static/src/js/wysiwyg/plugin/media.js [hunk 5]
```diff
@@ -1195,12 +1196,23 @@ var HandlePlugin = Plugins.handle.extend({
         }).data('target', $target); // save current target element.
 
         var src = $target.attr('src');
-        var sizingText = imageSize.w + 'x' + imageSize.h;
+        var displayInfo = imageSize.w >= 170 || (imageSize.w >= 120 && imageSize.h >= 58) || (imageSize.w >= 80 && imageSize.h >= 76);
```
Comment (VincentSchippefilt) [inline addons/web_editor/static/src/js/wysiwyg/plugin/media.js]: rename displayInfo to isLarge ? canHoldText ? shouldDisplay

**Comment by VincentSchippefilt** [comment]
robodoo r+

**Comment by VincentSchippefilt** [comment]
robodoo rebase-ff

**Pull request closed without merging**